include/re_http.h

```diff
@@ -182,6 +182,8 @@ struct http_conn;
 typedef void (http_req_h)(struct http_conn *conn, const struct http_msg *msg,
 			  void *arg);
 
+int http_listen_fd(struct http_sock **sockp, re_sock_t fd, http_req_h *reqh,
+		   void *arg);
```

Comment:
is re_sock_t used in the api ?

should we consider moving the definition to re_types.h ?

Reply:
Done: https://github.com/baresip/re/pull/600/commits/ee74f20808435e6d122a65b9a0c98bfcf258ae7d